docs/pages/management/operations/upgrading.mdx

```diff
@@ -53,9 +53,18 @@ Back up before upgrading. We have more instructions in [Backing up Teleport](./b
 <TabItem scope={["enterprise", "oss"]} label="Self-Hosted">
 When upgrading a single Teleport cluster:
 
+<Notice type="tip">
+Teleport requires upgrading the Auth Service from one major version to another in sequence.
```

Comment:
This makes it sound like the auth service has to Go from 8-9-10 etc, but isn't totally clear about the nodes.

Upgrading auth from 8 to 9 and then from 9 to 10 without upgrading other agents is going to break things. Can we improve the wording here so that it's clear that the entire cluster needs to be upgraded one major at a time rather than just the auth server?

Reply:
thanks @zmb3 , please review the latest language